tag_manager/authorized_api/src/tags/schema/tag_templates/video_html5_attributes.json

```diff
@@ -0,0 +1,91 @@
+            "minimum": 0,
+            "maximum": 100
+        },
+        "dimension_id": {
+            "description": "Custom dimension id",
+            "type": "integer",
+            "minimum": 1,
+            "maximum": 200
+        },
+        "template_options_element_selectors": {
+            "description": "Video element CSS selectors",
+            "type": "array",
+            "items": {
+                "$ref": "../tag_types.json#/definitions/non_empty_string"
+            },
+            "minItems": 1
+        },
+        "template_options_title_attribute": {
+            "description": "Used to store the name of the HTML attribute of the video element that you want to track as the custom event name",
+            "$ref": "../tag_types.json#/definitions/non_empty_string"
+        },
+        "template_options_track_thresholds": {
+            "description": "Percentage thresholds that track how far a visitor viewed a video. All values passed must be unique and within [0 .. 100] range.",
+            "type": "array",
+            "items": {
+                "$ref": "#/definitions/threshold"
+            },
+            "minItems": 1,
+            "uniqueItems": true
+        },
+        "template_options_volume_dimension_id": {
+            "$ref": "#/definitions/dimension_id"
+        },
+        "template_options_create_edit": {
+            "description": "[Tag template options](#tag-template-options)",
+            "type": "object",
+            "properties": {
+                "element_selectors": {
+                    "$ref": "#/definitions/template_options_element_selectors"
+                },
+                "title_attribute": {
+                    "$ref": "#/definitions/template_options_title_attribute"
+                },
+                "track_thresholds": {
+                    "$ref": "#/definitions/template_options_track_thresholds"
+                },
+                "volume_dimension_id": {
+                    "$ref": "#/definitions/template_options_volume_dimension_id"
+                }
+            },
+            "additionalProperties": false
+        },
+        "template_options_get": {
+            "description": "[Tag template options](#tag-template-options)",
+            "type": "object",
+            "properties": {
+                "element_selectors": {
+                    "$ref": "#/definitions/template_options_element_selectors"
+                },
+                "title_attribute": {
+                    "$ref": "#/definitions/template_options_title_attribute"
+                },
+                "track_thresholds": {
+                    "$ref": "#/definitions/template_options_track_thresholds"
+                },
+                "volume_dimension_id": {
+                    "$ref": "#/definitions/template_options_volume_dimension_id"
+                }
+            },
+            "additionalProperties": false,
+            "required": [
+                "element_selectors"
```

Comment:
This looks like a leftover from returning `video` as element selectors when there was no input from the user. Removing this would simplify template options for HTML5. Same is true for https://github.com/PiwikPRO/PPMS-TagManager-ApiDocs/blob/master/v1/tags/schema/tag_templates/video_html5_attributes.json